supertokens/userIdMapping.go

```diff
@@ -0,0 +1,217 @@
+package supertokens
+
+import (
+	"errors"
+)
+
+var userIdTypeDefault = "ANY"
+
+type CreateUserIdMappingResult struct {
+	OK                              *struct{}
+	UnknownSupertokensUserIdError   *struct{}
+	UserIdMappingAlreadyExistsError *struct {
+		DoesSuperTokensUserIdExist bool
+		DoesExternalUserIdExist    bool
+	}
+}
+
+func CreateUserIdMapping(supertokensUserId string, externalUserId string, externalUserIdInfo *string, force bool) (CreateUserIdMappingResult, error) {
```

Comment:
force bool should be optional with default value as false.

Reply:
Done